server/src/auth/auth.service.ts

```diff
@@ -1,14 +1,20 @@
 import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
-import { JwtService } from '@nestjs/jwt';
-import * as bcrypt from 'bcrypt';
-
+import * as bcrypt from 'bcryptjs';
 import { UsersService } from '../users/users.service';
 import { User } from '../users/entities/user.entity';
+<<<<<<< HEAD
+<<<<<<< HEAD
+import { jwtConstants } from './constants';
+=======
+import { JwtService } from '@nestjs/jwt';
+>>>>>>> updated import paths in user and auth service
+=======
 import { jwtConstants } from './constants';
+>>>>>>> merge confilct fix
```

Comment:
Can you remove these, we missed some when rebasing

Reply:
This looks like jwtConstants is actually in use, I can't run the server without it.

Reply:
he meant remove the `>>>>>`, `=======`, and `<<<<<<` lines
these are the merge conflict lines
lmk if you need help resolving